src/Migrations/Version20200413133226.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Kimai time-tracking app.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
```

Comment:
please use [this base class](https://github.com/kevinpapst/kimai2/blob/master/src/Doctrine/AbstractMigration.php):

```php
use App\Doctrine\AbstractMigration; 
```